### PR TITLE
Add getItem() to SlimefunItemStack

### PR DESCRIPTION
**Pull request by WalshyDev**
## Description
Add getItem() to SlimefunItemStack

## Changes
Added a nullable getItem() to SlimefunItemStack

## Related Issues
N/A

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [X] I followed the existing code standards and didn't mess up the formatting.
- [X] I did my best to add documentation to any public classes or methods I added.

### CODE REVIEW

File: src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java [hunk 1]
```diff
@@ -150,6 +151,15 @@ public String getItemID() {
         return id;
     }
 
+    /**
+     * Gets the {@link SlimefunItem} associated for this SlimefunItemStack. Null if no item is found.
+     *
+     * @return The {@link SlimefunItem} for this SlimefunItemStack, null if not found.
+     */
+    public SlimefunItem getItem() {
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java]: I'd probably even name it "getSlimefunItem()".
Also why is this build failing?

File: src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java [hunk 2]
```diff
@@ -150,6 +151,15 @@ public String getItemID() {
         return id;
     }
 
+    /**
+     * Gets the {@link SlimefunItem} associated for this SlimefunItemStack. Null if no item is found.
+     *
+     * @return The {@link SlimefunItem} for this SlimefunItemStack, null if not found.
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java]: Nitpicky but you could also mark "SlimefunItemStack" as a link, even though its the same file...

File: src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java (hunk 1 above)
Comment (WalshyDev) [reply src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java]: Very stupid things, fixed now. 

The reason I didn't name it that is, this is already an ItemStack and getting the ID is getItemID. Follows the pattern of using getItem. 

Personally, it should be both changed or keeping this like so.

File: src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java (hunk 2 above)
Comment (WalshyDev) [reply src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java]: Yeah... reee

**Pull request merged**